redis/redis_test.go

```diff
@@ -0,0 +1,60 @@
+package redis
+
+import (
+	"log"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/alicebob/miniredis"
+	"github.com/elliotchance/redismock"
```

Comment:
why do we need 2 different mock servers?

Reply:
`miniredis` is redis mock server and the `redismock` is lib to mock the client. 
Let me try to mock only the server.